ui-tests/cypress/support/pageObjects/content/management/AddPage.js

```diff
@@ -0,0 +1,82 @@
+import {htmlElements} from "../../WebElement";
+import Content from "../../app/Content.js";
+import AppPage from "../../app/AppPage.js";
+import ManagementPage from "./ManagementPage";
+
+export default class AddPage extends Content {
+
+  alertMessageDiv = `.alert`;
+  contentDescriptionInput = `${htmlElements.input}#description`;
+  contentTitleAttrInput = `.RenderTextInput`
+  contentAttrsItTab = `${htmlElements.a}#content-attributes-tabs-tab-it`;
+  contentTitleAttrInputIt = `attributes[0].values.it`
+  contentSaveButton = `${htmlElements.button}#saveopts`;
+  contentSaveButtonUl = `${htmlElements.ul}`;
+  contentSaveButtonWrapper = `${htmlElements.div}.StickySave__row--top`;
+  contentSaveButtonSaveAction = `${htmlElements.li}`;
+  contentAttrEnPane = `${htmlElements.div}#content-attributes-tabs-pane-en`;
+  contentAttrItPane = `${htmlElements.div}#content-attributes-tabs-pane-it`;
+
+  getAlertMessage() {
```

Comment:
if the alert is at the root level, maybe we should move it to the AppPage; is it used by other pages?

Reply:
agree, I will move it. Yes its under root. It will be needed for other pages too

Reply:
please, check the implementation done by @alepintus for the categories as he already created something similar (maybe even the same object)

Reply:
refactored, now using Alessios helper functions